mk/re.mk

```diff
@@ -233,7 +233,7 @@ ifeq ($(OS),linux)
 	CFLAGS		+= -fPIC -DLINUX
 	LIBS		+= -ldl
 	LFLAGS		+= -fPIC
-	SH_LFLAGS	+= -shared
+	SH_LFLAGS	+= -shared -Wl,-soname,libre.so.$(VER_ABI)
```

Comment:
re.mk is a shared makefile that can also be used by other projects.

Reply:
I overlooked that, is fixed now.